plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/SettingsForm.js

```diff
@@ -125,6 +128,24 @@ Ext.define('NX.controller.SettingsForm', {
   },
 
   /**
+   * Discard any changes to this form
+   *
+   * @private
+   */
+  discardChanges: function (button) {
+    var form = button.up('form'),
+        record = form.getRecord();
+
+    if (form.api && form.api.load) {
+      form.fireEvent('load', form);
+    }
+    else {
+      form.getForm().reset();
+      form.isValid();
```

Comment:
Not sure I understand. This is the handler for the discard button on forms. It used to exist as a handler in the view, but I moved it to the controller as a listener (which is useful, so forms can define custom discard buttons and just reuse this logic).

<img width="206" alt="repositories - sonatype nexus 2015-07-22 11-34-13" src="https://cloud.githubusercontent.com/assets/186715/8829659/d470db76-3065-11e5-90df-890b815fb391.png">